packages/node/package.json

```diff
@@ -5,7 +5,7 @@
     "lavamoat-run-command": "src/run-command.js"
   },
   "version": "7.1.0",
-  "main": "index.js",
+  "main": "src/index.js",
```

Comment:
I don't know how, but it worked at least once. Maybe by importing via direct path.